### PR TITLE
Update data-classification-content-explorer.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the delay due to crawl on Custom SIT

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10023

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 1]
```diff
@@ -96,6 +96,9 @@ Content explorer shows a current snapshot of the items that have a sensitivity l
 
 A [DLP policy](dlp-learn-about-dlp.md) can help protect sensitive information, which is defined as a **sensitive information type**. Microsoft 365 includes [definitions for many common sensitive information types](sensitive-information-type-entity-definitions.md) from across many different regions that are ready for you to use. For example, a credit card number, bank account numbers, and national ID numbers.
 
+> [!NOTE]
+> Custom Sensitive information types could take upto 48 hours due to crawl.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
> Custom sensitive information types could take up to 48 hours to appear due to crawl.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
confirming with PG

**Comment by chrfox** [comment]
The requested change is incorrect, closing without merging

**Pull request closed without merging**